recipes/dmalloc/dmalloc_5.5.2.oe

```diff
@@ -0,0 +1,60 @@
+DESCRIPTION = "Debug Malloc Library"
+HOMEPAGE = "http://dmalloc.com"
+LICENSE = "CC-BY-SA-3.0"
+
+SRC_URI = "\
+  http://dmalloc.com/releases/dmalloc-${PV}.tgz \
+  file://configure-pagesize-HACK.patch \
```

Comment:
Fixes the same issue, but with a decent description and less intrusive:
http://crosstool-ng.org/hg/crosstool-ng/file/tip/patches/dmalloc/5.5.2/110-block_size.patch

The other patch is needed with libc/compilers that optimizes str(n)dup into a macro that must be undefined before special cased. Maybe I should update the patch with a comment?